.storybook/preview.ts

```diff
@@ -74,20 +74,35 @@ setThemeContext({
   },
 } as StoreResponseType);
 
-async function configureThemeSettings() {
-  const module = (await import(`@/assets/presets/coffee.json`)) as {
-    default: IThemeConfigPreset;
-  };
-  const preset = module.default;
-
-  if (preset) {
-    const styleElement = document.createElement("style");
-    styleElement.innerText = ":root {";
-    Object.entries(preset).forEach(([key, value]) => {
-      styleElement.innerText += `--${key.replace(/_/g, "-")}: ${value};`;
-    });
-    styleElement.innerText += "}";
-    document.head.prepend(styleElement);
+// List of available theme presets
+const PRESETS = ["default", "coffee", "black-gold", "mercury", "purple-pink", "watermelon"] as const;
```

Comment:
The preset list is hardcoded and duplicates the keys from `/client-app/assets/presets/index.ts`. Better to import it and reuse.